README.md

```diff
@@ -197,11 +197,12 @@ to your configure options.
 
 #### Python
 
-GenTen provides Python bindings through a Python module called pygenten generated by [pybind11](https://github.com/pybind/pybind11).  This can be enabled by adding the configuration option
+GenTen provides Python bindings through a Python module called pygenten generated by [pybind11](https://github.com/pybind/pybind11) and requires shared libraries to be built by `CMake`.  The bindings can be enabled by adding the configuration option
 ```
  -D ENABLE_PYTHON=ON \
+ -D BUILD_SHARED_LIBS=ON \
```

Comment:
So this is actually only true on Linux.  On Windows and Mac, you do not need to enable shared libraries.